http-api/src/main/java/net/runelite/http/api/RuneLiteAPI.java

```diff
@@ -56,7 +56,12 @@
 			properties.load(in);
 
 			version = properties.getProperty("runelite.version");
-			rsVersion = Integer.parseInt(properties.getProperty("rs.version"));
+			String strRsVersion = properties.getProperty("rs.version");
+			if ("${rs.version}".equals(strRsVersion))
```

Comment:
You should probably catch NumberFormatException instead and rethrow RuntimeException with some message, or use the logger as already used there.

Reply:
Why?

Reply:
Why not, the string comparison for ${rs.version} looks ugly, and the NumberFormatException is less prone to change in the runelite.properties.

Reply:
No, I wanted a `java.lang.VerifyError` because by classpath is not in a valid state, so I should throw a relevant exception. Though it should probably be a `IncompatibleClassChangeError`

Reply:
I see, so I would throw it based on the NumberFormatException still.